test/rules/space-within-parens/default/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
please rename the folder to `nospace` or something like that since there is no default for this rule